ttnn/cpp/ttnn/common/constants.hpp

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+
+#pragma once
+
+namespace ttnn {
+
+constexpr uint8_t DefaultQueueId = 0;
```

Comment:
Now that this `DefaultQueueId` has been decontextualized from where it was previously defined (inside a function), I think it should be recontextualized with some brief description. There are lots of queues in our SW stack so it would be nice to know which type this is a default for :)

Reply:
You are right. 
I also want this to become a type next, instead of just some uint8_t

Reply:
That would have been the ideal... but baby steps :)

Reply:
I will craft a note in the next PR. Want to merge this to not accumulate conflicts